src/main/java/org/altbeacon/beacon/BeaconManager.java

```diff
@@ -138,27 +126,184 @@ public static void setDebug(boolean debug) {
     /**
      * The default duration in milliseconds spent not scanning between each bluetooth scan cycle when no ranging/monitoring clients are in the foreground
      */
-    public static final long DEFAULT_BACKGROUND_BETWEEN_SCAN_PERIOD = 5*60*1000;
+    public static final long DEFAULT_BACKGROUND_BETWEEN_SCAN_PERIOD = 5 * 60 * 1000;
 
     private long foregroundScanPeriod = DEFAULT_FOREGROUND_SCAN_PERIOD;
     private long foregroundBetweenScanPeriod = DEFAULT_FOREGROUND_BETWEEN_SCAN_PERIOD;
     private long backgroundScanPeriod = DEFAULT_BACKGROUND_SCAN_PERIOD;
     private long backgroundBetweenScanPeriod = DEFAULT_BACKGROUND_BETWEEN_SCAN_PERIOD;
 
+    private static Logger sLogger = Loggers.empty();
```

Comment:
I do prefer that  the androidLogger be the default.  Even if folks aren't running with BuildConfig.DEBUG, I'd like warnings and errors to show up in LogCat even if people have not configured a logger.